core/src/main/scala/org/apache/spark/scheduler/TaskInfo.scala

```diff
@@ -54,7 +52,7 @@ class TaskInfo(
    * accumulable to be updated multiple times in a single task or for two accumulables with the
    * same name but different IDs to exist in a task.
    */
-  val accumulables = ListBuffer[AccumulableInfo]()
```

Comment:
In order to maintain binary compatibility, I could rewrite this to be a `lazy val` that returns a `ListBuffer` formed from the "real" `accumulables` which can remain private. I might go ahead and do that just to avoid any chance of incompatibility-related problems, although I don't anticipate this being an issue in practice.